frontend/src/main/pages/AdminLoadSubjectsPage.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+
+
+import { useBackend } from "main/utils/useBackend";
+const AdminLoadSubjectsPage = () => {
+
+    
+    return (
+        <BasicLayout>
+            <h2>LoadSubjects</h2>
```

Comment:
Could we add a space in "Load Subjects" here? This page title will be seen by the user, so we should using proper English spacing here.